Controller/PostController.php

```diff
@@ -23,9 +23,11 @@
 class PostController extends Controller
 {
     /**
+     * @param Request $request
+     *
      * @return RedirectResponse
      */
-    public function homeAction()
+    public function homeAction(Request $request = null)
```

Comment:
Yes, that's true, but when extending this controller you could easily access the request attribute if needed.